packages/datadog-plugin-pg/src/index.js

```diff
@@ -32,9 +32,30 @@ function createWrapQuery (tracer, config) {
       }
 
       const originalCallback = pgQuery.callback
-      const statement = pgQuery.text
+      const statement = (pgQuery.cursor && pgQuery.cursor.text) || pgQuery.text
       const params = this.connectionParameters
 
+      function finishSpan (err) {
```

Comment:
This function and `isReadableStream` should be defined outside of the current function to avoid creating new instances on every call.